### PR TITLE
[W-16713324] migrate to Coveo salesforce-prod org

### PR DESCRIPTION
**Pull request by cheungaryk**
ref: W-16713324

- migrate to Coveo salesforce-prod org
- localize "version" in the version facets for JP site

### CODE REVIEW

**Review by jdomingu — DISMISSED** [review]
Not a dealbreaker, but added a comment.

Just FYI I'm seeing something really wonky when I try to test this so I can't verify locally unfortunately.

I swapped the Coveo keys in my `.zshrc` for the new prod keys, then went to this URL:
<http://localhost:8080/jp/general/search.html#q=test>

That shows this:
Screenshot 2024-09-18 at 10 54 13 AM

If you didn't see this in your testing, then I think you can ignore it.

File: src/js/21-coveo-facets.js [hunk 1]
```diff
@@ -26,7 +26,8 @@
         versionFacet.setAttribute('facet-id', `${componentID}-version`)
         versionFacet.setAttribute('field', 'mulesoftversionwithlatest')
         versionFacet.setAttribute('heading-level', 2)
-        versionFacet.setAttribute('label', `${componentDisplayName} Version`)
+        const label = isJapaneseSite() ? 'バージョン' : 'Version'
```
[the remark this answers is not in the document]
Comment (cheungaryk) [reply src/js/21-coveo-facets.js]: good call - I forgot about it. Just made the change and tested and it works!

**Comment by cheungaryk** [comment]
> Not a dealbreaker, but added a comment.
> 
> Just FYI I'm seeing something really wonky when I try to test this so I can't verify locally unfortunately.
> 
> I swapped the Coveo keys in my `.zshrc` for the new prod keys, then went to this URL: http://localhost:8080/jp/general/search.html#q=test
> 
> That shows this: Screenshot 2024-09-18 at 10 54 13 AM
> 
> If you didn't see this in your testing, then I think you can ignore it.

Sorry forgot to mention that we haven't promoted my changes to prod on Coveo, so you won't see anything yet. But if you use a non-prod API key, I think you will see the results (even with the salesforce prod org ID).

**Review by jdomingu — APPROVED** [review]

**Comment by jdomingu** [comment]
Nice. Thanks for taking the time to do that (and sorry that I forgot until just now to check back!)

**Pull request merged**